autoshop/api/v1/serializers.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import serializers
+
+from core.models import Profile
+
+
+class UserProfileSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Profile
+        fields = '__all__'
```

Comment:
I think we dont need time stamp model fields here. Also user should not be able to edit username which we use for login.